packages/docs/src/page-configs/ui-elements/chip/page-config.ts

```diff
@@ -66,4 +67,8 @@ const config: ApiDocsBlock[] = [
   block.api(VaChip, apiOptions),
 ]
 
+if (table) {
```

Comment:
If the component has a “variables" file, then we add such a block to the end of the documentation, otherwise why add an empty block?

Reply:
The question only about conditional block `is (table) {`
I don't see a point of why it's being used at all.

@ishpartko thanks for responding :D.

Reply:
Ohhh, I see. As I remember it was in the old implementation and left because before "table" was the result of a function call, now it's just import of existing file.

Merry Christmas to you @asvae!

Reply:
@asvae now fixed